Queuesim/Server.py

```diff
@@ -1,17 +1,22 @@
+import simpy
+
 from abc import ABC, abstractmethod
 
 class ServerBase(ABC):
 
-    def __init__(self, env, resource):
+    def __init__(self, env, capacity=1):
         self.__env = env
-        self.__resource = resource
```

Comment:
I think bringing the resource creation inside the server is good. It avoids potential mismatch of env. However, I think it's a bad idea defaulting the capacity.